doc/api/stream.md

```diff
@@ -535,8 +535,10 @@ added: v8.0.0
 
 * Returns: `this`
 
-Destroy the stream, and emit the passed error. After this call, the
-writable stream has ended. Implementors should not override this method,
+Destroy the stream, and emit the passed error and a close event.
```

Comment:
Should error and close be in backticks? It looks like they are, in the next section.